test/unit/cameras/PerspectiveCamera.js

```diff
@@ -0,0 +1,56 @@
+/**
+ * @author simonThiele / https://github.com/simonThiele
+ */
+
+module( "PerspectiveCamera" );
+
+test( "updateProjectionMatrix", function() {
+	var near = 1,
+			far = 3,
+			bottom = -1,
+			top = 1,
```

Comment:
Okay, no problem thanks for reviewing all this stuff. In my understanding, none of these values (l, r, t, b, n, f) depending on fov since they are defining the boundings of the near clipping plane. The bounding of the far clipping plane are depending on fov ( www.songho.ca/opengl/gl_projectionmatrix.html )